wiki/People/Completionists/en.md

```diff
@@ -0,0 +1,30 @@
+# Completionists
+
+Completionists are players who take on the extremely daunting task of playing **every [Ranked](/wiki/Beatmap/Category#ranked) [beatmap](/wiki/Beatmap)** of a given game mode. With the amount of Ranked beatmaps increasing every day, this feat is becoming progressively more difficult to achieve, requiring insane amounts of dedication and perseverance.
```

Comment:
you'd want to minimise this kind of fluff in the wiki

```suggestion
**Completionists** are players who have completed every [Ranked](/wiki/Beatmap/Category#ranked) [beatmap](/wiki/Beatmap) of a given [game mode](/wiki/Game_mode). Because the amount of Ranked beatmaps increases every day, this feat becomes progressively more difficult to achieve.
```